platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -46,6 +46,28 @@ The embedded Kotlin has been updated to link:https://github.com/JetBrains/kotlin
 
 === Deprecations
 
+[[string_invoke]]
+==== Referencing tasks and domain objects by `"name"()` in Kotlin DSL
+
+In Kotlin DSL, it is possible to reference a task or other domain object by its name using the `"name"()` notation.
+This isn't a very useful construct and link:https://github.com/gradle/gradle/issues/27699[causes hard to understand conflicts].
```

Comment:
The phrase is a little negative and out of place. I suggested we move it to the bottom.
```suggestion
```